appcues/src/main/java/com/appcues/ui/utils/AppcuesPaddings.kt

```diff
@@ -0,0 +1,63 @@
+package com.appcues.ui.utils
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.LayoutModifier
+import androidx.compose.ui.layout.Measurable
+import androidx.compose.ui.layout.MeasureResult
+import androidx.compose.ui.layout.MeasureScope
+import androidx.compose.ui.platform.InspectorInfo
+import androidx.compose.ui.platform.InspectorValueInfo
+import androidx.compose.ui.platform.debugInspectorInfo
+import androidx.compose.ui.unit.Constraints
+import androidx.compose.ui.unit.constrainHeight
+import androidx.compose.ui.unit.constrainWidth
+import androidx.compose.ui.unit.offset
+
+@Stable
+internal fun Modifier.appcuesPaddings(paddingValues: PaddingValues) =
```

Comment:
havent studied details - would it be possible at all to just coerce to zero but still use the default `.padding` for this one - delegate into it? or do we need to reimplement here.

Reply:
will look into it

Reply:
did you get to see the changes I've made?